solo-chains/runtime/starlight/src/lib.rs

```diff
@@ -521,31 +521,14 @@ impl pallet_treasury::Config for Runtime {
     type WeightInfo = ();
     type SpendFunds = ();
     type SpendOrigin = TreasurySpender;
-    type AssetKind = VersionedLocatableAsset;
-    type Beneficiary = VersionedLocation;
+    type AssetKind = ();
+    type Beneficiary = AccountId;
     type BeneficiaryLookup = IdentityLookup<Self::Beneficiary>;
-    type Paymaster = PayOverXcm<
-        TreasuryInteriorLocation,
-        crate::xcm_config::XcmRouter,
-        crate::XcmPallet,
-        ConstU32<{ 6 * HOURS }>,
-        Self::Beneficiary,
-        Self::AssetKind,
-        LocatableAssetConverter,
-        VersionedLocationConverter,
-    >;
-    type BalanceConverter = UnityOrOuterConversion<
```

Comment:
All of these changes are strictly needed for the treasury pallet to only allow spends in the local native token? Or is it another reason for them?

Reply:
only for that, yes

Reply:
disregard